src/sentry/api/serializers/models/integration.py

```diff
@@ -19,10 +19,75 @@ def serialize(self, obj, attrs, user):
             'provider': {
                 'key': provider.key,
                 'name': provider.name,
-            }
+            },
         }
 
 
+class IntegrationConfigSerializer(IntegrationSerializer):
+    def serialize(self, obj, attrs, user):
+        data = super(IntegrationConfigSerializer, self).serialize(obj, attrs, user)
+
+        data.update({
+            'config_organization': [],
+            'config_project': [],
+        })
+
+        try:
+            install = obj.get_installation()
+            data.update({
```

Comment:
are you expecting anything in this update to raise? if not, i think it would be nicer to move this block into an `else`

Reply:
00360e5